content/tutorials/metrics.md

```diff
@@ -0,0 +1,18 @@
++++
+title= "Metric Monitoring"
+date= 2018-03-13T10:23:10-07:00
+description = ""
+draft= false
++++
+
+In starting Clipper v0.3.0, Clipper report its metrics to a prometheus server. 
```

Comment:
Change this sentence to: "Starting with version 0.3.x, Clipper reports metrics to a Prometheus server"

Add a link to the Prometheus documentation here as well.